src/Player/States/Air.gd

```diff
@@ -3,21 +3,27 @@ extends PlayerState
 
 
 func physics_process(delta: float) -> void:
+	var velocity_previous_y = _parent.velocity.y
```

Comment:
Please use type hints. Here, you can use type inference `var velocity_previous_y := _parent.velocity.y`

If I get it correctly, the changes here prevent the player from sliding up steep slopes in the Air state? If so, this is not the way to handle this. When walking against a slope that's too steep, the player should not enter the Air state in the first place.

Reply:
I was being a bit lazy, inference wasn't working in this case. I'll fix it.